docs/website/docs/plus/features/transformations/executing.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Execute transformations
+description: Define and execute local transformations
+---
```

Comment:
Afaik the cache at the moment only works with data loaded to the filesystem destination, so running transformations in the project context using the cache will only work on filesystem inputs.

Reply:
Done